dlc_developer_config.toml

```diff
@@ -31,10 +31,10 @@ benchmark_mode = false
 [build]
 # Add in frameworks you would like to build. By default, builds are disabled unless you specify building an image.
 # available frameworks - ["autogluon", "huggingface_tensorflow", "huggingface_pytorch", "huggingface_tensorflow_trcomp", "huggingface_pytorch_trcomp", "pytorch_trcomp", "tensorflow", "mxnet", "pytorch", "stabilityai_pytorch"]
-build_frameworks = []
+build_frameworks = ["stabilityai_pytorch"]
 
 # By default we build both training and inference containers. Set true/false values to determine which to build.
-build_training = true
+build_training = false
```

Comment:
quick heads up, once builds are fixed, you'll want to set sagemaker_remote_tests to "standard" in order to run the SM tests you're adding in the PR.

Would also suggest setting datetime_tag to false while iterating